osbrain/address.py

```diff
@@ -4,6 +4,7 @@
 from ipaddress import ip_address
 
 import zmq
+import os
```

Comment:
Imports from the standard library should go first (i.e.: at the top of the file).